### PR TITLE
Add 'Chip' component

### PR DESCRIPTION
**Pull request by Ebazhanov**
-------------------
This PR is related to adding a new component `Chip` -> Jira ticket [CONTACT-529](https://comatch.atlassian.net/browse/CONTACT-529)
- provided by [design](https://app.zeplin.io/project/5a291fa3dd015b9e085b67e2/screen/5f7eda0971c08355f8d26779)

Preview: [story](https://github.com/COMATCH/comatch-design-system/blob/ebf94cceb71b62856fa7954d24df463d0505cd84/src/components/stories/Chip.stories.tsx#L27-L27)

-------------------

### CODE REVIEW

File: src/components/Chip/styled.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    font-size: ${({ theme: { typography } }) => typography.body.fontSize};
+    font-weight: normal;
+    text-align: center;
+
+    &.reverse {
+        flex-direction: row-reverse;
+
+        .Icon {
+            margin-left: ${({ theme: { spacing } }) => spacing(2)};
+            margin-right: ${({ theme: { spacing } }) => spacing(0)};
```
Comment (stanimir-stoychev) [inline src/components/Chip/styled.ts:19]: If you're using 0 then there's no need to use `spacing`

File: src/components/Chip/styled.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+const Chip = styled.div`
+    border-radius: 12px;
+    border: ${({ theme: { palettes } }) => `1px solid ${palettes.greyScale.main.bgColor}`};
+    display: flex;
+    background-color: ${({ theme: { palettes } }) => palettes.greyScale.lighter.bgColor};
+    max-width: 350px;
+    padding: ${({ theme: { spacing } }) => `0 ${spacing(2)}`};
+    font-size: ${({ theme: { typography } }) => typography.body.fontSize};
+    font-weight: normal;
```
Comment (stanimir-stoychev) [inline src/components/Chip/styled.ts:11]: I believe this is a default value. Please make sure it is and if so then just delete the line

File: src/components/Chip/Chip.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+function Chip(props: ComponentProps) {
+    const { children, disabled, icon, reverse } = props;
+
+    if (!children) {
+        return null;
+    }
+
+    return (
+        <StyledChip className={classnames('Chip', { disabled, reverse })}>
```
Comment (stanimir-stoychev) [inline src/components/Chip/Chip.tsx:15]: What about `id`, `className`, and `generateCss` coming from the props?

File: src/components/Chip/styled.ts [hunk 4]
```diff
@@ -0,0 +1,33 @@
+import styled from 'styled-components';
+
+const Chip = styled.div`
+    border-radius: 12px;
+    border: ${({ theme: { palettes } }) => `1px solid ${palettes.greyScale.main.bgColor}`};
```
Comment (stanimir-stoychev) [inline src/components/Chip/styled.ts:5]:
```suggestion
    border: 1px solid;
    border-color: ${({ theme: { palettes } }) => palettes.greyScale.main.bgColor};
```

**Review by stanimir-stoychev — CHANGES_REQUESTED** [review]

File: src/components/Chip/styled.ts (hunk 1 above)
Comment (Ebazhanov) [reply src/components/Chip/styled.ts]: thx, updated

File: src/components/Chip/styled.ts (hunk 2 above)
Comment (Ebazhanov) [reply src/components/Chip/styled.ts]: thx, updated

File: src/components/Chip/Chip.tsx (hunk 3 above)
Comment (Ebazhanov) [reply src/components/Chip/Chip.tsx]: thx, updated

File: src/components/Chip/styled.ts (hunk 4 above)
Comment (Ebazhanov) [reply src/components/Chip/styled.ts]: thx, updated

**Review by stanimir-stoychev — APPROVED** [review]
LGTM

**Pull request merged**